mail/views.py

```diff
@@ -73,6 +74,26 @@ def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
         )
 
 
+class AutomaticEmailView(GenericAPIView):
```

Comment:
If I use the `ListAPIView` would that make it more complicated to add a method to handle a `PATCH` request for a single record later?